extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -111,6 +111,7 @@ export let InteractiveReactData;
  *    mutedState: boolean,
  *    pageAudioState: boolean,
  *    pageHasElementsWithPlaybackState: boolean,
+ *    panningMediaState: {x: string, y: string, zoom: string},
```

Comment:
I suspect the store will trigger the `PANNING_MEDIA_STATE` update even when nothing has changed, if you set it to the same value.
Can you write a comparisonFunction in this file? That should be very straightforward and ensure the callbacks only get called on change.